src/map/rodex.c

```diff
@@ -395,7 +395,8 @@ void rodex_read_mail(struct map_session_data *sd, int64 mail_id)
 	nullpo_retv(sd);
 
 	msg = rodex->get_mail(sd, mail_id);
-	nullpo_retv(msg);
+	if (msg == NULL)
```

Comment:
probably better add exclude condition for not show nullpo error for example if mail_id is zero or other what can get from non official clients?
because nullpo can catch unknown errors.